tests/cases/article/@article-type.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../jats.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../../../support.sch" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?expected-error message="@article-type is ignored."?>
```

Comment:
Since the attribute is a node, we should be able to include a `node`/`line`. But the `location` returned in the SVRL doesn't include proper XPath (so no details of _which_ element the attribute is on):

```xml
<svrl:schematron-output xmlns:svrl="http://purl.oclc.org/dsdl/svrl" title="" schemaVersion="">
    
    <svrl:active-pattern id="attribute-whitelist" name="attribute-whitelist"/>

    <svrl:fired-rule context="@*"/>
    <svrl:failed-assert test="name()='xml:lang'" location="/@article-type">
        <svrl:text>
            @article-type is ignored.
        </svrl:text>
    </svrl:failed-assert>

    <svrl:active-pattern id="element-whitelist" name="element-whitelist"/>

    <svrl:fired-rule context="*"/>
    <svrl:fired-rule context="*"/>
    <svrl:failed-assert test="name()='article'" location="/article/front">
        <svrl:text>
            &lt;front&gt; is ignored.
        </svrl:text>
    </svrl:failed-assert>

    <svrl:fired-rule context="*"/>
    <svrl:failed-assert test="name()='article'" location="/article/front/article-meta">
        <svrl:text>
            &lt;article-meta&gt; is ignored.
        </svrl:text>
    </svrl:failed-assert>

</svrl:schematron-output>
```

Reply:
Opened https://github.com/Schematron/schematron/pull/81.